author-service/src/main/resources/application.yml

```diff
@@ -15,3 +15,8 @@ spring:
     hibernate:
       ddl-auto: update
     database: postgresql
+
+eureka:
+  client:
+    service-url:
+      default-zone: http://localhost:8761/eureka
```

Comment:
move into ${HOST}

Reply:
This bug is fixed, see: [AuthorMicroservice](https://github.com/hizmailovich/AuthorMicroservice) and [BookMicroservice](https://github.com/hizmailovich/BookMicroservice).